python/ct/crypto/verify.py

```diff
@@ -211,24 +333,15 @@ def verify_sct(self, sct, chain):
             ct.crypto.error.EncodingError: failed to encode signature input,
                 or decode the signature.
             ct.crypto.error.SignatureError: invalid signature.
+            ct.crypto.error.MissingCertificateError: a certificate is missing.
```

Comment:
In this case, how about IncompleteChainError? that would be more indicative of the error.